src/main/java/seedu/address/logic/commands/TrackCommand.java

```diff
@@ -3,41 +3,45 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
-import seedu.address.logic.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.CategoryContainsKeywordsPredicate;
+import seedu.address.model.person.CategoryContainsKeywordPredicate;
 
 /**
  * Tracks and lists all persons in the address book who are in the specified category to the user.
  */
 public class TrackCommand extends Command {
 
     public static final String COMMAND_WORD = "track";
-    public static final String MESSAGE_ARGUMENTS = "Category: %1$s";
+    public static final String MESSAGE_INVALID_INPUT_ERROR = COMMAND_WORD
```

Comment:
Thanks for making the error message clearer so that the users know what they are supposed to input